### ISSUE TITLE
Pass encryption key into CheckDatabaseEncryptionKey

### ISSUE THREAD

**Issue by x2bool** [egramtel/egram.tel#20]
CheckDatabaseEncryptionKey takes encryption key as a parameter. Database should be encrypted and the key somehow should be stored on computer (OS-specific key storage???).

**Comment by worldbeater** [comment]
Speaking of local storage, personally I would advice either of those: <a href="https://github.com/realm/realm-dotnet">Realm</a>, <a href="https://github.com/mbdavid/LiteDB">LiteDB</a>.

**Comment by x2bool** [comment]
Yeah we are gonna need some DB. I am up to anything really. I didn't try Realm, but it looks cool.
